code/controllers/subsystem/lobotomy_corp.dm

```diff
@@ -222,6 +228,25 @@ SUBSYSTEM_DEF(lobotomy_corp)
 		return meltdown_occured
 
 /datum/controller/subsystem/lobotomy_corp/proc/RollOrdeal()
+	if(GLOB.master_mode == "suppression")
```

Comment:
Very ugly, but it works, I guess?
Would really prefer if you could find a way to shove it all into gamemode's code instead.